src/main/scala/com/fulcrumgenomics/umi/FilterConsensusReads.scala

```diff
@@ -28,6 +28,7 @@ import com.fulcrumgenomics.FgBioDef._
 import com.fulcrumgenomics.bam.Bams
 import com.fulcrumgenomics.cmdline.{ClpGroups, FgBioTool}
 import com.fulcrumgenomics.umi.VanillaUmiConsensusCallerOptions._
+import com.fulcrumgenomics.util.NumericTypes.PhredScore
```

Comment:
One of the CLP params is of type PhredScore.  We had at some point introduced extra type defs for PhredScore in the consensus caller objects, which I've now eliminated, making this import necessary.